cpp/PdBase.hpp

```diff
@@ -24,7 +24,10 @@
 
 // define this to use C++11 std::mutex for locking
 #ifdef LIBPD_USE_STD_MUTEX
-    #if __cplusplus < 201103L
+	#ifdef _WIN32
+		// __cplusplus is badly behaved in windows. just assume we have mutex
```

Comment:
I don;t like this. Why just *assume*? Maybe someone is using libpd with an older C++ library and can't compile it with C++11 support?